src/andromede/libs/standard_sc.yml

```diff
@@ -24,6 +24,226 @@ library:
         - name: emission
 
   models:
+    - id: link_with_storage
+      description: A link with energy storage
+      parameters:
+        - name: f_from_max
+          time-dependent: false
+          scenario-dependent: false
+        - name: f_to_max
+          time-dependent: false
+          scenario-dependent: false
+        - name: capacity
+          time-dependent: false
+          scenario-dependent: false
+        - name: initial_level
+          time-dependent: false
+          scenario-dependent: false
+      variables:
+        - name: r
+          lower-bound: 0
+          upper-bound: capacity
+        - name: f_from
+          lower-bound: -f_from_max
+          upper-bound: f_from_max
+        - name: f_to
+          lower-bound: -f_to_max
+          upper-bound: f_to_max
+        - name: f_from_p
+          lower-bound: 0
+        - name: f_from_m
+          lower-bound: 0
+        - name: f_to_p
+          lower-bound: 0
+        - name: f_to_m
+          lower-bound: 0
+      ports:
+        - name: flow_from
+          type: flow
+        - name: flow_to
+          type: flow
+        - name: flow_from_pos
+          type: flow
+        - name: flow_to_pos
+          type: flow
+      port-field-definitions:
+        - port: flow_from
+          field: flow
+          definition: -f_from
+        - port: flow_to
+          field: flow
+          definition: f_to
+      constraints:
+        - name: max0_from
+          expression: f_from = f_from_p - f_from_m
+        - name: max0_to
+          expression: f_to = f_to_p - f_to_m
+        - name: r_t1
+          expression: r[t+1] = r[t] + f_from - f_to
+        - name: r0
+          expression: r[0] = initial_level
+        - name: rt
```

Comment:
`final_level`

There is a problem here ! How to specify the level of final time of the block in the yaml and even in our lower-level expressions @sylvlecl, I think it is not possible for now ?